listenbrainz/db/dump_manager.py

```diff
@@ -121,3 +123,41 @@ def import_dump(private_archive, public_archive, listen_archive, threads):
     except Exception as e:
         log.error('Unexpected error while importing data: %s', str(e))
         raise
+
+
+@cli.command()
+@click.argument('location', type=str)
+def delete_old_dumps(location):
+    _cleanup_dumps(location)
+
+
+def _cleanup_dumps(location):
+    """ Delete old dumps while keeping the latest two dumps in the specified directory
+
+    Args:
+        location (str): the dir which needs to be cleaned up
+
+    Returns:
+        (int, int): the number of dumps remaining, the number of dumps deleted
+    """
+    dump_re = re.compile('listenbrainz-dump-[0-9]*-[0-9]*')
+    dumps = [x for x in sorted(os.listdir(location), reverse=True) if dump_re.match(x)]
+    if not dumps:
+        print('No dumps present in specified directory!')
+        return
+
+    keep = dumps[0:2]
```

Comment:
We're keeping only the latest two dumps right now (like in MB), but I can change this to take an argument, if we want to tinker with how many dumps we want to keep on ftp simultaneously.

Reply:
It would be nice to have it defined as a constant at the top of the file for clarity,but no need to make it a config item.